style-guide.rst

```diff
@@ -0,0 +1,628 @@
+
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  Click the button that says, "Save."
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  Click :guilabel:`Save`.
+  
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  You may see an error message that says, "Something went wrong."
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  You may get an error: ``Something went wrong.``
+
+  
+.. _straight-quote:
+  
+Straight quotes
+~~~~~~~~~~~~~~~~~~
+
+Any time that you *do* need to use quotation marks, use straight (or *plain*) quotes. Sphinx and Docutils will output the typographically correct quote style.
+
+.. _serial-comma:
+
+Serial comma
+-----------------
+
+In a comma-delineated list of items, the penultimate item should be followed by a comma.
+
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  Apples, oranges and pears.
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  Apples, oranges, and pears.
+  
+A bulleted list is often more clear than an inline list.
+
+.. rubric:: Correct
+
+.. code-block:: rst
+
+  You will need to be familiar with git, GitHub, and Python.
+  
+.. rubric:: Possibly Better
+
+.. code-block:: rst
+
+  You will need to be familiar with:
+  
+  - git
+  - GitHub
+  - Python
+  
+There's no hard rule about which to use in any situation. Use your judgement: try it both ways and see which is more clear.
+
+.. _avoid-unneeded-words:
+
+Avoid unneeded words
+-----------------------
+
+.. _adverbs:
+
+Adverbs
+~~~~~~~~~~~
+
+Adverbs often contribute nothing.
+
+Common offenders include:
+
+ - simply
+ - easily
+ - just
+ - very
+ - really
+ - basically
+
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  To open the file, simply click the button.
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  To open the file, click the button.
+  
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  You can easily edit the form by...
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  To edit the form...
+  
+.. _filler-phrases:  
+  
+Filler words and phrases
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Many words and phrases provide no direct meaning. They are often inserted to make a sentence seem more formal, or to simulate a perceived style of business communication. These should be removed.
+
+Common filler phrases and words include:
+
+- to the extent that
+- for all intents and purposes
+- when all is said and done
+- from the perspective of
+- point in time
+
+This list is not exhaustive. These "canned phrases" are pervasive in technical writing. Remove them whenever they occur.
+
+.. _semicolons:
+
+Semicolons
+-------------
+
+Semicolons are used to separate two independent clauses which could stand as individual sentences but which the writer feels would benefit by close proximity.
+
+Semicolons can almost always be replaced with periods (full stops). This rarely diminishes correctness and often improves readability.
+
+.. rubric:: Correct
```

Comment:
Sometimes you use Right, other times you use Correct. Please standardize!

Reply:
LAWD what was I doing?!?!

Reply:
OH! I REMEMBER WHAT I WAS DOING

I used "Right" vs. "Wrong"
but...
"(Technically) Correct" vs. "Improved"